source/administration/object-management/transition-objects-to-azure.rst

```diff
@@ -148,6 +169,17 @@ The example above uses the following arguments:
        remote storage tier. Specify the name in all-caps, e.g. ``AZURE_TIER``.
        This value is required in the next step.
 
+   * - :mc-cmd:`ACCOUNT <mc ilm tier add --account-name>`
```

Comment:
@ravindk89 `account-name` and `account-key` flags correspond to Azure Blob storage account. ref: https://learn.microsoft.com/en-us/azure/storage/common/storage-account-overview#storage-account-name

Reply:
Aha, good catch.